README.md

```diff
@@ -224,12 +224,26 @@ Environment:
              -e JWT_PUBLIC_KEY=xxxxxx \
              -e DELIUS_ENDPOINT_URL=https://ndseis.ad.nps.internal/api
              -e DELIUS_API_USERNAME=xxxxxxx \
-             -e APPLICATION_INSIGHTS_KEY=xxxxx \
+             -e APP_INSIGHTS_KEY=xxxxx \
              -d -t mojdigitalstudio/community-proxy:latest`
 
 [ For PRODUCTION - supply all values - no defaults ]
 
 
+# Run Locally (outside of Docker)
+
+Supply the information required (detailed in the run.sh, which docker uses)
+
+java -DAPPLICATION_INSIGHTS_IKEY=xxxxx \
```

Comment:
wouldn't specify the app insights key when running locally otherwise you will pollute t2 / t3 environments unnecessarily

Reply:
ok. I'll move it out of the example but put a note in that this is how you would do it to test that events are being received. Thanks.